Remove deprecation notice from e2e readmes (#34476)
woocommerce/woocommerce: plugins/woocommerce/tests/e2e-pw/README.md

### Introduction

> End-to-end tests are powered by Playwright. The test site is spinned up using `wp-env` (recommended), but we also temporarily support `e2e-environment`.

Reviewer: Should we update the wording on `temporarily` to be something like below?

```suggestion
End-to-end tests are powered by Playwright. The test site is spinned up using `wp-env` (recommended), but we will continue to support `e2e-environment` in the meantime.
```

Author: I think thats a good idea
